runtime/doc/options.txt

```diff
@@ -1322,6 +1322,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	The value of this option is stored with the tab page, so that each tab
 	page can have a different value.
 
+	When 'cmdheight' is zero, it disables echo area and all outputs needs
+	|hit-enter| prompts.
```

Comment:
That doesn't seem desirable. Can be improved later I guess.

Reply:
It is improved by https://github.com/neovim/neovim/pull/16480.